src/components/Rockets.js

```diff
@@ -8,7 +8,7 @@ import { fetchingData } from '../redux/rockets/rocketsReducer';
 import store from '../redux/configureStore';
 
 const Rockets = () => {
-  const rocketsFromStore = useSelector((store) => store.rocketsReducer);
```

Comment:
Just for you to take into account, in this line, in the arrow function inside the `useSelector` hook, it doesn't matter the name you put for your variable, for this case is the same. You can even assign some shorthand expressions like this:
`  const rocketsFromStore = useSelector((e) => e.rocketsReducer) `

Reply:
Thanks for the good info I was confused somehow when I try different names it works perfectly but I get the point now.

Thanks!